docs/resources/bcs_instance.md

```diff
@@ -0,0 +1,379 @@
+  Changing this will create a new instance.
+
+* `sfs_turbo` - (Optional, List, ForceNew) Specifies the information about the SFS Turbo file system.
+  Changing this will create a new instance.
+  The sfs_turbo object structure is documented below.
+
+* `sign_algorithm` - (Optional, String, ForceNew) Specifies the secutity mechanism used by the BCS instance.
+  Valid values are `ECDSA` and `SM2`(Chinese cryptographic algorithms, The basic and professional don't support this
+  algorithm). Default is `ECDSA`.
+  Changing this will create a new instance.
+
+* `tc3_need` - (Optional, Bool, ForceNew) Specified whether to add Trusted computing platform.
+  Changing this will create a new instance.
+
+The `cluster_info` block supports:
+
+* `id` - (Required, String, ForceNew) Specifies the unique ID of the CCE cluster.
+  Changing this creates a new instance.
+
+* `name` - (Required, Int, ForceNew) Specifies the name of the CCE cluster.
+  Changing this creates a new instance.
+
+-> **NOTE:** CCE only supports version 1.15 and below.
+
+The `peer_orgs` block supports:
+
+* `org_name` - (Required, String, ForceNew) Specifies the name of the peer organization.
+  Changing this creates a new instance.
+
+* `channel` - (Required, String, ForceNew) Specifies the channel name to which the peer organization is bound.
+  Changing this creates a new instance.
+
+* `count` - (Required, Int, ForceNew) Specifies the number of peers in organization.
+  Changing this creates a new instance.
+
+The `channels` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the name of the channel.
+  Changing this creates a new instance.
+
+The `couchdb` block supports:
+
+* `user_name` - (Required, String, ForceNew) Specifies the user name of the couch datebase.
+  Changing this creates a new instance.
+
+* `password` - (Required, String, ForceNew) Specifies the password of the couch datebase.
+  The password consists of 8 to 26 characters and must consist at least three of following: uppercase letters,
+  lowercase letters, digits, special charactors(!@$%^-_=+[{}]:,./?).
+  Changing this creates a new instance.
+
+The `sfs_turbo` block supports:
+
+* `availability_zone` - (Optional, String, ForceNew) Specifies the availability zone in which to create the SFS turbo.
+  Please following [reference](https://developer.huaweicloud.com/en-us/endpoint/?all) for the values.
+  Changing this creates a new instance.
+
+* `flavor` - (Optional, Int, ForceNew) Specifies the flavor of SFS turbo.
+  Changing this creates a new instance.
+
+* `share_type` - (Optional, String, ForceNew) Specifies the share type of the SFS turbo.
+  Changing this creates a new instance.
+
+* `type` - (Optional, String, ForceNew) Specifies the type of SFS turbo.
+  Changing this creates a new instance.
+
+The `block_info` block supports:
+
+* `max_message_count` - (Optional, String, ForceNew) Specifies the number of transactions included in the block.
```

Comment:
the type of `max_message_count` should be int

Reply:
finish, other parameters has checked.
the parameters name of the `block_info` has been updated according to console.